### ISSUE TITLE
Windows Defender : Trojan : False Positive?

### ISSUE THREAD

**Issue by worthwhileindustries** [nanopool/XMR-stak#3]
![image](https://user-images.githubusercontent.com/3536254/38904440-979036ae-4267-11e8-8faa-cee0b5926c31.png)

**Comment by gazhayes** [comment]
Just go to the actual XMR-Stak repository where the source code is an compile it yourself (even if you've never done it before it's not *that* hard), don't touch this binary crap it could be anything.

**Comment by worthwhileindustries** [comment]
@gazhayes thanks is this the official repo? https://github.com/fireice-uk/xmr-stak is this the

**Comment by gazhayes** [comment]
Yeah that's the one I used anyway, I also went through the source and didn't find anything to raise suspicions.

I compiled on Linux but the Windows instructions are here: https://github.com/fireice-uk/xmr-stak/blob/master/doc/compile_Windows.md

It looks pretty straight forward, just follow it to the letter and it should work. If you run into problems search the open *and* closed issues on that repository and someone else has probably already asked.

**Comment by worthwhileindustries** [comment]
@gazhayes thanks for the info. so much garbage out there in cryptoland.
